src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -1,11 +1,8 @@
 #include <subsystems/IntakeSubsystem.h>
 
-IntakeSubsystem::IntakeSubsystem() {
-  // m_rightIntakeSpinnyBoy.Follow(m_leftIntakeSpinnyBoy, false);
-}
+IntakeSubsystem::IntakeSubsystem() {}
```

Comment:
This could be removed from here and also removed from the header since it's doing nothing